pkg/cmd/server/origin/master.go

```diff
@@ -377,11 +381,19 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 
 	imageStorage := imageetcd.NewREST(c.EtcdHelper)
 	imageRegistry := image.NewRegistry(imageStorage)
+	imageStreamSecretsStorage := imagesecret.NewREST(c.ImageStreamSecretClient())
 	imageStreamStorage, imageStreamStatusStorage, internalImageStreamStorage := imagestreametcd.NewREST(c.EtcdHelper, imagestream.DefaultRegistryFunc(defaultRegistryFunc), subjectAccessReviewRegistry)
 	imageStreamRegistry := imagestream.NewRegistry(imageStreamStorage, imageStreamStatusStorage, internalImageStreamStorage)
 	imageStreamMappingStorage := imagestreammapping.NewREST(imageRegistry, imageStreamRegistry)
 	imageStreamTagStorage := imagestreamtag.NewREST(imageRegistry, imageStreamRegistry)
 	imageStreamTagRegistry := imagestreamtag.NewRegistry(imageStreamTagStorage)
+	importerFn := func() imageimporter.Interface {
+		return imageimporter.NewImageStreamImport(c.Options.ImagePolicyConfig.MaxImagesBulkImportedPerRepository, util.NewTokenBucketRateLimiter(2.0, 3))
+	}
+	importerDockerClientFn := func() dockerregistry.Client {
+		return dockerregistry.NewClient(20*time.Second, false)
```

Comment:
Same value we used before.  Very short dial times are abusive to users because if the server is legitimately slow (goes through a proxy) and it's right on the edge it makes the behavior seem flaky.  Also, conference wifi can easily take >30s to dial in some cases - it's gotten better over the past few years, but still not great.  Should be a tunable down the road but I'm being judicious until we hit the problem.